src/excmds.ts

```diff
@@ -426,19 +426,52 @@ export function urlparent (count = 1){
 /**
  * Open a URL made by modifying the current URL
  *
- * @param mode      -t text replace
- *                  -r regexp replace
- *                  -q replace the value of the given query
- *                  -Q delete the given query
- *                  -g graft a new path onto URL or parent path of it
- * @param replacement the replacement arguments:
- *                  -t <old> <new>
- *                  -r <regexp> <new> [flags]
- *                  -q <query> <new_val>
- *                  -Q <query>
- *                  -g <graftPoint> <newPathTail>
- *                      - graftPoint > 1 to count from left
- *                      - graftPoint < 0 to count from right
+ * There are several modes:
+ *
+ * * Text replace mode:   `urlmodify -t <old> <new>`
+ *
+ *   Replaces the first instanct of the text `old` with `new`.
+ *      * `http://example.com` -> (`-t exa peta`) -> `http://petample.com`
+ *
+ * * Regex replacment mode: `urlmodify -r <regexp> <new> [flags]`
+ *
+ *   Replaces the first match of the `regexp` with `new`. You can use
+ *   flags `i` and `g` to match case-insensitively and to match
+ *   all instances respectively
+ *      * `http://example.com` -> (`-r [ea] X g`) -> `http://XxXmplX.com`
+ *
+ * * Query replace mode: `urlmodify -q <query> <new_val>`
+ *
+ *   Replace the value of a query with a new one:
+ *      * `http://e.com?id=foo` -> (`-q id bar`) -> `http://e.com?id=bar
+ *
+ * * Query delete mode: `urlmodify -Q <query>`
+ *
+ *   Deletes the given query (and the value if any):
+ *      * `http://e.com?id=foo&page=1` -> (`-Q id`) -> `http://e.com?page=1`
+ *
+ * * Graft mode: `urlmodify -g <graft_point> <new_path_tail>`
+ *
+ *   "Grafts" a new tail on the URL path, possibly removing some of the old
+ *   tail. `graft_point` >= 0 counts path levels from the left, < 0 counts
```

Comment:
As I mentioned on Matrix, I think this needs a little more explanation. You could say that you are counting from starting from the left of the path, and perhaps have website.com/this/is/a/path?

Additionally, it might be worth mentioning that "-1" is a bit magic in that you append an extra level to the path before you start counting.